code/perception/src/vision_node.py

```
@@ -2,30 +2,21 @@
 import cv2
-from vision_node_helper import get_carla_class_name, get_carla_color
+from vision_node_helper import coco_to_carla, carla_colors
 from rospy.numpy_msg import numpy_msg
 from sensor_msgs.msg import Image as ImageMsg
-from std_msgs.msg import Header, Float32MultiArray
+from std_msgs.msg import Float32MultiArray
 from cv_bridge import CvBridge
-from torchvision.utils import draw_bounding_boxes, draw_segmentation_masks
+from torchvision.utils import draw_segmentation_masks
 import numpy as np
 from ultralytics import NAS, YOLO, RTDETR, SAM, FastSAM
```

Ruff (0.8.2)

`ultralytics.NAS` imported but unused

Remove unused import

(F401)